### PR TITLE
Task/historical data

### PR DESCRIPTION
**Pull request by cvenen**

### CODE REVIEW

File: reuse23/README.md [hunk 1]
```diff
@@ -1,56 +1,100 @@
...
+- `run.py` is for running experiments
+- `summarize.py` is for uploading data into the database
+
+## Build and Update Engines
+1. First consult the `ENGINELOG.md` located in this directory for instructions on how to build each engine. 
+2. Create an `engines.json` file in the directory in which the engines are built and populate it with the first build information. 
+3. To check and update the engines, run the `build.py` script in the home directory.
+
+### How to run `build.py`
+This script should be ran in the home directory and `engines.json` must exist with a dictionary for each engine and its build. The script will attempt to build all engines listed in `common.py` in the `assign_engines()` function.
```
Comment (titzer) [inline reuse23/README.md:14]: Small typo: s/ran/run

What do you mean by "home" directory? Is that the root of the wish-you-were-fast repo or this reuse23 directory?

File: reuse23/README.md [hunk 2]
```diff
@@ -1,56 +1,100 @@
...
+```
+2. Run the script with the following command:
+```
+python3 wish-you-were-fast/reuse23/build.py
+```
+3. Manually update `ENGINELOG.md` with required information (date, version, etc.).
+
+
+## Run Experiments
+1. After building engines, make sure symlinks are updated in the `reuse23/engines` directory. Navigate to `wish-you-were-fast/reuse23/engines` and excute the following command:
```
Comment (titzer) [inline reuse23/README.md:28]: s/excute/execute

File: reuse23/README.md [hunk 3]
```diff
@@ -1,56 +1,100 @@
...
+```
+# Replacing a symlink
+ln -sf ~/<engine path> ./<engine name-version>-link
+
+# Creating a new symlink
+ln -s ~/<engine path> ./<engine name-version>-link
+```    
+### How to run `run.py`
+This script should be ran in the home directory and btime must be installed. Running experiments on specific configs must be changed in `common.py` manually.
```
Comment (titzer) [inline reuse23/README.md:37]: s/ran/run again

**Review by titzer — COMMENTED** [review]
lgtm with some minor cleanups

File: reuse23/README.md (hunk 1 above)
Comment (cvenen) [reply reuse23/README.md:14]: We meant to refer to ~/ as the "home" directory (the directory where the wish-you-were-fast repo was cloned)

**Pull request merged**